Modules/PowerShellHelp.psm1

```diff
@@ -0,0 +1,112 @@
+<#
+.SYNOPSIS
+
+Write-DscResourcePowerShellHelp generates PowerShell compatable help files for a DSC
+resource module
+
+.DESCRIPTION
+
+The Write-DscResourcePowerShellHelp cmdlet will review all of the MOF based resources
+in a specified module directory and will inject PowerShell help files for each resource
+in to the specified directory. These help files include details on the property types for
+each resource, as well as a text description and examples where they exist.
+
+.PARAMETER OutputPath
+
+Where should the files be saved to (ususally the "en-US" folder inside the module for US english)
+
+.PARAMETER ModulePath
+
+The path to the root of the DSC resource module (where the PSD1 file is found, not the folder for
+and individual DSC resource)
+
+.EXAMPLE 
+
+This example shows how to generate help for a specific module
+
+    Write-DscResourcePowerShellHelp -ModulePath C:\repos\SharePointdsc -OutputPath C:\repos\SharePointDsc\en-US
+
+#>
+function Write-DscResourcePowerShellHelp
```

Comment:
`Write` is communication verb.  You are creating a file (resource.) `New` would be the more appropriate verb.